schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -691,16 +691,13 @@
             "type": "array",
             "description": "This is problem type information (e.g. CWE identifier). Must contain: At least one entry, can be text, OWASP, CWE, please note that while only one is required you can use more than one (or indeed all three) as long as they are correct). (CNA requirement: [PROBLEMTYPE])",
```

Comment:
I'm not sure why there is a cweID but not a owaspId and couldn't remember where we landed on that. Otherwise, I think this one description is relatively straightforward.

Reply:
ok to leave OWASP mentions, no need for a field.